src/mon/MonClient.cc

```diff
@@ -111,14 +107,17 @@ int MonClient::get_monmap_privately()
 
   ldout(cct, 10) << "have " << monmap.epoch << " fsid " << monmap.fsid << dendl;
 
+  std::random_device rd;
```

Comment:
let's use random_r() to avoid seeding issues with this (http://www.pcg-random.org/posts/cpps-random_device.html)